Example/Program.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Cimpress.Auth0.Client;
+using Cimpress.Clients.Foma.Model;
+using Cimpress.Extensions.Http.MessageHandlers;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace Cimpress.Clients.Foma.Example
+{
+    class Program
+    {
+        private readonly FomaClient fomaSdk;
+        private readonly ILogger logger;
+
+        static void Main(string[] args)
+        {
+            if (args.Length != 2)
+            {
+                Console.Write("You must specify 2 arguments: <username> <password>");
+                return;
+            }
+
+            try
+            {
+                new Program(args[0], args[1]).Execute().Wait();
+            }
+            catch (Exception ex)
+            {
+                Console.Write(ex);
+            }
+        }
+
+        public Program(string username, string password)
+        {
+            // logger factory / adjust to your needs to log to the desired destination
+            var loggerFactory = new LoggerFactory();

```

Comment:
I've now attached the console that's provided built-in by `Microsoft.Extensions.Logging.Console`, and added various log messages. Additionally, I ended the execution with a `Console.ReadLine()` so people see the result instead of getting it closed after a quick execution.